page.codeberg.gwync.glerm.yml

```diff
@@ -0,0 +1,33 @@
+id: page.codeberg.gwync.glerm
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: glerm
+finish-args:
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+
+modules:
+  - pypi-deps.json
+  - name: glerm
+    buildsystem: simple
+    build-commands:
+      - install -Dm 644 page.codeberg.gwync.glerm.metainfo.xml /app/share/metainfo/page.codeberg.gwync.glerm.metainfo.xml
+      - install -Dm0644 icon.png /app/share/icons/hicolor/256x256/apps/page.codeberg.gwync.glerm.png
+      - install -Dm 644 page.codeberg.gwync.glerm.desktop /app/share/applications/page.codeberg.gwync.glerm.desktop
+    sources:
+      - type: file
+        url: https://codeberg.org/gwync/glerm/raw/commit/5ae0af67609441b6e8aff05e9dcbd7c4f5e01787/page.codeberg.gwync.glerm.metainfo.xml
+        dest-filename: page.codeberg.gwync.glerm.metainfo.xml
+        sha256: c19a5185deebb29de5895a104625b4e8914077e8c1e3e1bf9843a4fe17c6d396
```

Comment:
screenshot link needs to be raw link not to the html page. https://codeberg.org/gwync/glerm/raw/commit/a77978332dab876b11fb8799298a19c9922f70bc/screenshot.png

Once you fix that, update commit and sha256 here.